templates/gnark/sindri.json

```diff
@@ -0,0 +1,9 @@
+{
+  "circuit_type": "gnark",
+  "name": "{{ packageName }}",
```

Comment:
"name" is the Sindri API circuit name. I chose to make this the same as the gnark/go package name for now, but maybe we want to have that be a separate question

Reply:
I'd strongly prefer for this to be a separate question and to use the `circuitName` here. I had added this under the common questions because it applies to all circuits, and set up the validation logic to match the backend and for it to pull from the directory name in the same way that a git repo or npm package would match the directory name. The validation logic for the package name is not the same as for circuit names, and the `kebab-case` that is a common convention for these sort of names is not allowed for package names. Note that I *use* the circuit name to generate the default for the package name, taking care to remove unsupported characters, so it's generally as simple as hitting enter if you don't want to customize the package name.

Reply:
My mistake - I saw the "circuitName" field in your initial `Sindri.json` template and assumed you meant `circuitStructName` (the specific gnark api variable), not the `name` field in the Sindri.json structure.